books/free-programming-books-zh.md

```diff
@@ -484,6 +484,7 @@
 * [命名函数表达式探秘](http://justjavac.com/named-function-expressions-demystified.html) - kangax、为之漫笔(翻译) (原始地址无法打开，所以此处地址为justjavac博客上的备份)
 * [你不知道的Javascript](https://github.com/getify/You-Dont-Know-JS/tree/1ed-zh-CN)
 * [深入理解JavaScript系列](http://www.cnblogs.com/TomXu/archive/2011/12/15/2288411.html)
+* [现代 JavaScript 教程](https://zh.javascript.info)
```

Comment:
Who is its instructor? Ilya Kantor?

Please complete metainfo acording to CONTRIBUTING.MD guidelines.